cli/rollup.config.js

```diff
@@ -81,7 +81,13 @@ export default [
     },
     plugins: [
       ...plugins,
-      execute(`chmod +x dist/${cliConfig.name}`),
+      {
```

Comment:
Can you contribute this into the CLI seed as well please.

Reply:
good riddance!